server/src/test/java/org/elasticsearch/indices/IndicesRequestCacheTests.java

```diff
@@ -80,14 +85,14 @@ public void testBasicOperationsCache() throws Exception {
         // cache hit
         entity = new TestEntity(requestCacheStats, indexShard);
         loader = new Loader(reader, 0);
-        value = cache.getOrCompute(entity, loader, reader, termBytes);
+        value = cache.getOrCompute(entity, loader, mappingKey, reader, termBytes);
         assertEquals("foo", value.streamInput().readString());
         assertEquals(1, requestCacheStats.stats().getHitCount());
         assertEquals(1, requestCacheStats.stats().getMissCount());
         assertEquals(0, requestCacheStats.stats().getEvictions());
         assertTrue(loader.loadedFromCache);
         assertEquals(1, cache.count());
-        assertTrue(requestCacheStats.stats().getMemorySize().bytesAsInt() > value.length());
+        assertTrue(requestCacheStats.stats().getMemorySize().getBytes() > value.length());
```

Comment:
This isn't strictly required but the old method is deprecated and I figured I'd clean it up while I was here.